docs/ui/components/Sidebar/SidebarCollapsible.tsx

```diff
@@ -60,16 +61,16 @@ export function SidebarCollapsible(props: Props) {
 
   return (
     <>
-      <a css={titleStyle} onClick={toggleIsOpen}>
+      <ButtonBase css={titleStyle} aria-expanded={isOpen ? 'true' : 'false'} onClick={toggleIsOpen}>
         <div css={chevronContainerStyle}>
           <ChevronDownIcon
             size={iconSize.tiny}
             css={[chevronStyle, !isOpen && chevronClosedStyle]}
           />
         </div>
         <CALLOUT weight="medium">{info.name}</CALLOUT>
```

Comment:
This seems to be rendered a _paragraph_ `p` element, but you probably don't want a paragraph inside a button, because that's not valid HTML. `button` can only contain [phrasing content](https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element).